### PR TITLE
Fixes #24105 - Add gpg-key fingerprint for debian repositories

### PR DESCRIPTION
**Pull request by m-bucher**

### CODE REVIEW

**Comment by johnpmitsch** [comment]
@m-bucher can you update the PR title with the issue number? You can make it the same as the commit message

**Comment by m-bucher** [comment]
Dependency PR has been merged: pulp/pulp_deb#50

**Comment by sjha4** [comment]
[test katello]

**Comment by stbenjam** [comment]
@m-bucher Would you mind rebasing this PR? Thanks!

**Comment by m-bucher** [comment]
@stbenjam Rebased, thank you.

**Comment by stbenjam** [comment]
[test katello]

File: app/models/katello/glue/pulp/repo.rb [hunk 1]
```diff
@@ -220,6 +220,8 @@ def deb_importer_values(capsule)
           feed: self.importer_feed_url(capsule),
           releases: self.deb_releases,
           components: self.deb_components,
+          gpg_keys: (self.gpg_key ? self.gpg_key.content : nil),
+          require_signature: (not self.gpg_key.blank?),
```
Comment (stbenjam) [inline app/models/katello/glue/pulp/repo.rb]: `self.gpg_key.present?`

**Comment by m-bucher** [comment]
Thanks for the feedback @stbenjam 
Sorry, it took some time to address.

**Comment by sbernhard** [comment]
[test katello]

**Comment by sbernhard** [comment]
[test katello]

**Comment by stbenjam** [comment]
[test katello]

**Comment by stbenjam** [comment]
@m-bucher @sbernhard Are you ready for this to be merged?

**Comment by m-bucher** [comment]
@stbenjam as far as I am concerned, the code is ready to be merged. Dependencies in pulp_deb and runcible have already been merged.
I'd say got for it, thanks.

**Comment by stbenjam** [comment]
Thanks!

**Pull request merged**